Gopkg.toml

```diff
@@ -92,7 +92,7 @@ required = [
 
 [[constraint]]
   name = "k8s.io/helm"
-  version = "2.14.3"
+  version = "=2.14.3"
```

Comment:
dep wanted to use 2.15.2, so I locked the dependency; it will be upgraded to 2.16.1 via https://github.com/giantswarm/helmclient/pull/134